packages/dynamic-import-vars/README.md

```diff
@@ -0,0 +1,193 @@
+[npm]: https://img.shields.io/npm/v/@rollup/plugin-dynamic-import-vars
+[npm-url]: https://www.npmjs.com/package/@rollup/plugin-dynamic-import-vars
+[size]: https://packagephobia.now.sh/badge?p=@rollup/plugin-dynamic-import-vars
+[size-url]: https://packagephobia.now.sh/result?p=@rollup/plugin-dynamic-import-vars
+
+[![npm][npm]][npm-url]
+[![size][size]][size-url]
+[![libera manifesto](https://img.shields.io/badge/libera-manifesto-lightgrey.svg)](https://liberamanifesto.com)
+
+# @rollup/plugin-dynamic-import-vars
+
+🍣 A rollup plugin to support variables in dynamic imports in Rollup.
+
+Turns:
+
+```js
+function importLocale(locale) {
```

Comment:
So it's not entirely clear how/why the source is transformed here. It _is_ explained further down, but users will see this one first. Let's copy `Turns` and `Into` to around line 131. Remove the `Into` section here, and add `Supports concatonated string imports such as:` and then let's add the following beneath it: 

```md
For more information, please see [How It Works](#how-it-works)
```

That should be a good summary, and then folks can read more detail below.

Reply:
I moved it, but I kept a small code snippet so that at first glance people understand what "variables in dynamic imports" means.